js/PaperWallet.js

```diff
@@ -1,4 +1,40 @@
+import { newArray, newArray0, copyArray } from './utils/arrays';
+import { applyModule } from './utils/wasm';
+import { loadRustModule } from './RustModule';
+
+export const scramble = (module, iv, password, input) => {
+    if (iv.length != 4) {
+        throw new Error('IV must be 4 bytes');
+    }
+    const bufiv = newArray(module, iv);
+    const bufinput = newArray(module, input);
+    const bufpassword = newArray(module, password);
+    const bufoutput = newArray0(module, input.length + 4);
+    module.paper_scramble(bufiv, bufpassword, password.length, bufinput, input.length, bufoutput);
+    let result = copy_array(module, bufoutput, input.length + 4);
```

Comment:
`copy_array` will not work here, since it's now called `copyArray`

Reply:
good catch. looking forward to be able to test everything smoothly